zx.mjs

```diff
@@ -117,7 +118,8 @@ async function importPath(filepath, origin = filepath) {
   if (ext === '.ts') {
     let {dir, name} = parse(filepath)
     let outFile = join(dir, name + '.mjs')
-    await run`npm_config_yes=true npx tsc --target esnext --module esnext --moduleResolution node ${filepath}`
+    let tsc = await which('tsc').then(() => 'tsc', () => 'npm_config_yes=true npx tsc')
```

Comment:
What if user want to run tsc from node_nodules and have globally installed tsc?

Reply:
We may introduce `$.tsc` for this case.

Reply:
Not sure, but this case might be handled by `npx`. Afair, npx tries to find pkg locally, and if it's absent, then fetches from registry.

Reply:
We can check `node ./node_modules/.bin/tsc`.